test/test_config_parser.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require 'test/unit'
+require 'config_parser'
+require 'config_pb'
+require 'fluent/config/v1_parser'
+
+# tests for config tool
+class TestConfigParser < Test::Unit::TestCase
+  # helper function to create object of message Param
+  def get_param(name, value)
```

Comment:
group helper function as private methods. You can group them in the end of the class.

from an example of airbnb guideline: 
https://github.com/airbnb/ruby/blob/0eee7930195fa9bbef30e63068bf5d9d2720d2b7/README.md#access-modifiers

an example:
https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud/blob/bf8cc81d3f4b36014ee93edfb48e36cf7dbde542/test/plugin/test_out_google_cloud.rb#L393

Reply:
That's a good suggestion - I'll make that change!